server/cmd/mmctl/commands/user_e2e_test.go

```diff
@@ -938,7 +938,7 @@ func (s *MmctlE2ETestSuite) TestPromoteGuestToUserCmd() {
 		printer.Clean()
 
 		err := promoteGuestToUserCmdF(s.th.Client, nil, []string{user.Email})
-		s.Require().Nil(err)
+		s.Require().NotNil(err)
```

Comment:
Can we use `s.Require().Error(err)` here to make it clear that an error is expected?

Reply:
Yes, changed in the commit. BTW, do you think that in line 931, shall we change the ```s.Require().Nil(err)``` to ```s.Require().Error(err)``` as well?

Reply:
It should be `NoError`, right?

Reply:
> It should be `NoError`, right?

You mean in line 931, right?